network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html

```diff
@@ -50,7 +53,10 @@ <h1 class="h1-heading">{{ page.header }}</h1>
       <div class="container">
         <h2 class="h3-heading">{{ section.sectionHeader }}</h2>
         <p class="body-large mb-4">{{ section.sectionCopy }}</p>
-        <a href="{{ section.sectionButtonURL }}" class="btn btn-ghost">{{ section.sectionButtonTitle }}</a>
+        <a href="{{ section.sectionButtonURL }}" class="btn btn-ghost {% if section.sectionButtonURL2 and section.sectionButtonTitle2 %}mr-2 mb-2{% endif %}">{{ section.sectionButtonTitle }}</a>
```

Comment:
Can we put `ml-2 mt-2` on the second button instead of having to do this check?

Reply:
<img width="321" alt="image" src="https://user-images.githubusercontent.com/2896608/50786309-07e72b80-1268-11e9-8b57-8379e67da91e.png">

nope since it will misalign the 2 buttons when they can fit on the same row